Altis_Life.Altis/core/functions/fn_handleDamage.sqf

```diff
@@ -20,31 +20,27 @@ if (!isNull _source) then {
     if (_source != _unit) then {
         if (currentWeapon _source in ["hgun_P07_snds_F","arifle_SDAR_F"] && _projectile in ["B_9x21_Ball","B_556x45_dual"]) then {
             if (side _source isEqualTo west && playerSide isEqualTo civilian) then {
```

Comment:
if (side _source isEqualTo west && playerSide isEqualTo civilian) then {
to
if (side _source isEqualTo west && (playerSide isEqualTo civilian ||  {playerSide isEqualTo east}) then {